### DISCUSSION TITLE
When installing panel, why can't I git clone the code

### DISCUSSION THREAD

**Discussion by catalystsitaly** [pterodactyl/panel#3202] [Support]
I tried installing panel using git clone repository rather than released version. But it turns out not working. 
Why?

**Comment by parkervcp** [comment]
The development repo is not guaranteed to work unless you know what you are doing.

You can pull the latest release branch though.

**Reply by catalystsitaly** [reply]
yes, i git  checkout release/v1.3.1, panel shows 500 | server error. Tried on different vps all got the same result. Then I realize I have to download the zip in release. Problem solved.

**Reply by DaneEveritt** [reply]
You need to compile assets for it to work, you can't just clone and go.

**Reply by catalystsitaly** [reply]
got it, I used to clone and go in v0.7 so...thanks
